happi/tests/test_cli.py

```diff
@@ -121,6 +121,21 @@ def test_repair_name(runner: CliRunner, bad_happi_cfg: str):
     assert all([r['_id'] == r['name'] for r in res])
 
 
+def test_repair_no_save_if_no_changes(runner: CliRunner, happi_cfg: str):
+    # some arbitrary entry
+    search_string = '_id=tst_base_pim'
+
+    # run repair, which should find it doesn't neeod to repair the name and so doesn't need to save
+    runner.invoke(happi_cli, ['--path', happi_cfg, 'repair', search_string])
+
+    with search.make_context('search', [search_string], obj=happi_cfg) as ctx:
+        res = search.invoke(ctx)
+    last_edit_time = res[0]['last_edit']
+    # lets assume the starting values in happi_cfg stay the same,
+    # since other tests in this file also rely on it too.
+    assert last_edit_time == 'Thu Apr 12 14:40:08 2018'
```

Comment:
I'd like us to avoid hardcoding values here, on the off-chance the formatting changes in the future (or if we re-render the test database.  Could we instead search -> copy time -> repair -> search again -> verify no change

I'm sure I've hard-coded field info in this test suite in the past, I'm not free of sin here haha